src/math_parser_diag.cpp

```diff
@@ -49,6 +50,21 @@ std::function<void( dialogue &, double )> u_val_ass( char scope,
     }
 }
 
+std::function<double( dialogue & )> option_eval( char scope,
+        std::vector<std::string> const &params )
+{
+    return[option = params[0]]( dialogue const & d ) {
+        return get_option<float>( option );
+    };
+}
+
+std::function<void( dialogue &, double )> option_ass( char scope,
+        std::vector<std::string> const &/*params*/ )
+{
+    debugmsg( "assigning options is not currently supported in math" );
+    return []( dialogue const &, double ) {};
```

Comment:
```suggestion
    throw std::invalid_argument( "assigning options is not currently supported in math" );
```
Otherwise it passes validation in a bad PR. Or better yet, don't define the assignment function at all.